specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json

```diff
@@ -2736,6 +2747,32 @@
       },
       "description": "This is the storage profile of a Gallery Image Version."
     },
+    "ValidationsProfile": {
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "validationEtag": {
+          "type": "string",
+          "description": "The published time of the image version"
+        },
+        "executedValidations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExecutedValidation"
+          },
+          "x-ms-identifiers": [],
+          "description": "This specifies the list of executedValidations of the image version metadata"
+        },
+        "platformAttributes": {
+          "type": "object",
+          "additionalProperties": {
```

Comment:
We have a lintDiff error for the additionalProperties object, https://github.com/Azure/azure-rest-api-specs/pull/23507/checks?check_run_id=13668499518 . 
additionalProperties should only be used for user defined tags, https://github.com/Azure/azure-openapi-validator/blob/main/docs/avoid-additional-properties.md 

Does your use case fit that? If it does, the ARM team will need to review this specifically and approve this.

Reply:
it is a dictionary, same as the advancedSetting in this case:
https://github.com/Azure/azure-rest-api-specs/blob/b7026df140e026a4311fbdd6a56d3fd427376f2f/specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json#L2105

Reply:
@purbch , that doesn't answer my question. 
It seems like this is intended for very specific information, right? The Publisher and Offer and Sku and Version, right? That's not user defined tags I believe.